addons/mail/static/src/views/fields/many2one_avatar_user_noname/many2one_avatar_user_noname_field.xml

```diff
@@ -0,0 +1,7 @@
+<templates>
+    <t t-name="mail.Many2OneAvatarUserNoNameField" t-inherit="mail.Many2OneAvatarUserField">
+        <xpath expr="//Many2One" position="attributes">
+            <attribute name="cssClass">"d-none"</attribute>
```

Comment:
... really? A whole field for this? :smile: 

Can't we achieve the same result with just some extra css?